content/200-concepts/300-more/050-environment-variables/200-using-multiple-env-files.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Using multiple .env files'
+metaTitle: 'Using multiple .env files.'
+metaDescription: 'Learn how to setup a dedicated testing environment using multiple .env files.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+There is a risk that your production database could be deleted if you store different connection URLs to each of your environments within a single `.env` file.
```

Comment:
This should probably be clarified as it is suuuuper scary. When and how is this possible?